map/src/App.tsx

```diff
@@ -68,8 +70,14 @@ class App extends React.Component<Props, State> {
     this.setState({ updateResultsCallback: callback });
   };
 
-  private setSelectedResult = (selectedResult: MarkerIdAndInfo | null) => {
-    this.setState({ selectedResult });
+  private setSelectedResult = (
+    selectedResult: MarkerIdAndInfo | null,
+    markerClicked = false,
+  ) => {
+    this.setState({
+      selectedResult,
+      mapMarkerClicked: markerClicked,
```

Comment:
renamed this as `mapMarkerClicked` instead, because this state has a larger scope and needs to be accessed by the Map.tsx too.. 

Also the state for Accordion is handled inside Results.tsx, so i renamed the state to `expandAccordion` there for clarity (previously it was `expandResult`

Reply:
https://github.com/reach4help/reach4help/pull/1776#discussion_r841234717